webots-docker/webots-config.py

```diff
@@ -0,0 +1,15 @@
+import json, os, sys
+
+TEAM_NAME = os.environ.get('TEAM_NAME')
+ROBOT_HOSTS = os.environ.get('ROBOT_HOSTS')
+
+filename = sys.argv[1]
+
+with open(filename, 'r') as f:
+    game_config = json.load(f)
+
+if TEAM_NAME: game_config['red']['config'] = 'teams/' + TEAM_NAME + '.json'
```

Comment:
Ifs on the same line as their bodies is definitely not my jam haha

Reply:
I think this codebase needs to be blackened and isorted